### ISSUE TITLE
How is base url specified when generating absolute urls

### ISSUE THREAD

**Issue by aszenz** [BolZer/symfony-typescript-routes#5]
Thanks for making this library, I was looking for something like this for a long time.
I had some questions regarding the working of the library

I'm wondering how do u pick up the base url for generating absolute urls
And also the command throws error when any route doesn't have scheme defined
Is there an option to only generate relative url's?

**Comment by BolZer** [comment]
Hi @aszenz 

Sorry for the late response. Give me couble of hours and you will be able to generate only relatives or absolute urls. A new version will follow shortly.

Regarding the absolute routes: All information are attached at the route object. However there some edge cases like subdomains that are currently not covered. I'll probably change the code in such a way that everything that's necessary for the absolute urls, but not given at the route object level (like the scheme), will be pushed as required arguments for the typescript function.

**Comment by BolZer** [comment]
@aszenz 

Hi, just released version 1.1.0

` require bolzer/symfony-typescript-routes:1.1.0`

You may now generate only relative urls.

```php
// some_command.php
<?php

use Bolzer\SymfonyTypescriptRoutes\Service\GeneratorService;
use Symfony\Component\Console\Command\Command;
use Symfony\Component\Console\Input\InputInterface;
use Symfony\Component\Console\Output\OutputInterface;
use Bolzer\SymfonyTypescriptRoutes\Dto\GeneratorConfig;

class GenerationCommand extends Command
{
    public function __construct(
        private GeneratorService $generatorService,
    ){}

      protected function configure(): void
    {
        $this->setName('generate_paths');
    }

    protected function execute(InputInterface $input, OutputInterface $output): int
    {
       // Typescript routes containing relative and absolute urls.
       $routes = $this->generatorService->generate(GeneratorConfig::generateEverything());
       
       // Typescript routes containing absolute urls.
       $routes = $this->generatorService->generate(GeneratorConfig::generateOnlyAbsoluteUrls());
       
       // Typescript routes containing absolute urls.
       $routes = $this->generatorService->generate(GeneratorConfig::generateOnlyRelativeUrls());
    
       file_put_contents(__DIR__ . '../../../paths.ts', implode("\n", $routes));
       $output->writeln('<comment>Generation of paths done.</comment>');
       return 1;
    }
}
``

**Comment by aszenz** [comment]
Thanks for the quick update. 👍
